### PR TITLE
Channel priority integration test 2

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

**Comment by kalefranz** [comment]
@mcg1969 The failure here is on line 431 of the PR.  Can you review and see if you agree that the code being tested in in error, or if the test itself is wrong?

**Comment by kalefranz** [comment]
superseded by #2937

**Comment by kalefranz** [comment]
@mcg1969 I'd like to close this PR one way or another.  If the test itself is correct, then let's fix the failure.  Or if the test is wrong, then let's just close the PR...

File: tests/test_create.py [hunk 1]
```diff
@@ -404,6 +405,35 @@ def test_clone_offline_multichannel_with_untracked(self):
...
+            # pycosat should be in the SUPERCEDED list
+            superceded_split = update_stdout.split('SUPERCEDED')
+            assert len(superceded_split) == 2
+            assert 'pycosat' in superceded_split[1]
+
+            # python sys.version should show conda-forge python
+            sys_version = check_output('{0} -c "import sys; print(sys.version)"'
+                                       ''.format(join(prefix, PYTHON_BINARY)), shell=True)
+            sys_version = sys_version.decode('utf-8')
+            assert "packaged by conda-forge" in sys_version, sys_version
```
Comment (HugoTian) [inline tests/test_create.py]: @kalefranz , the test fails in this line 

    assert "packaged by conda-forge" in sys_version, sys_version

The python is still in default channel, seems we need to activate env to work. Maybe we can use the get_conda_list_tuple method, and check the 4th element is 'conda-forge'. Just like how we check the version of pycosat.

**Comment by HugoTian** [comment]
Please refer to PR #3306

**Pull request closed without merging**